### PR TITLE
Updated to Handle weekly leaderboard for now

### PR DESCRIPTION
**Pull request by thebadcoder96**

### CODE REVIEW

**Comment by thebadcoder96** [comment]
Right now this takes the last message in the channel and then checks if contains Weekly leaderboard and that it is under 24 hours; if so it skips and the action is logged.

This is fine now since we have a leaderboard channel where nobody messages (maybe we should lock it)

**Comment by meri-nova** [comment]
#30 - Issue

File: src/discord_app.py [hunk 1]
```diff
@@ -373,6 +377,19 @@ async def send_weekly_leaderboard():
     # TODO: run every hour, but send only after 24 hours from the last report
     # Leaderboard task only runs in production on Heroku
     logging.info('Sending weekly leaderboard')
+    channel = await client.fetch_channel(GENERAL_CHANNEL_ID)
+    #Get last message in channel
+    async for message in channel.history(limit=1):
+        last_message = message
+        break
+    if last_message and last_message.author == client.user:
```
Comment (meri-nova) [inline src/discord_app.py:385]: amazing job. if it doesn't work i will delete this entire repo.

**Review by meri-nova — APPROVED** [review]
amazing job. if it doesn't work i will delete this entire repo.

**Pull request merged**